docs/SETTING_UP_A_NEW_INSTANCE.md

```diff
@@ -0,0 +1,85 @@
+# Setting up a new instance of DARIA
+
+These are detailed instructions in spinning up an instance in heroku. These assume some familiarity with the ideas and concepts and don't go into much detail, such as where to click, the mechanics involved, etc.
+
+## By the way
+
+DCAF has a lot of the surrounding infrastructure set up in such a way that it's easy to add new instances to our set. So please reach out to us if you're interested in skipping a lot of the tedious technical setup here.
+
+## Main steps
+
+### Create a new heroku app
+
+* Create a new Heroku app (if in DCAF pipeline: named xxx-fundname)
```

Comment:
good flag on this! Refers to the heroku pipeline we have set up, but I'll make a tweak to this to clarify for sure.